tests/test_forms.py

```diff
@@ -0,0 +1,155 @@
+from django.test import TestCase
+from unittest import mock
+from taxi.forms import (
+    DriverCreationForm,
+    DriverLicenseUpdateForm,
+    DriverSearchForm,
+    CarForm,
+)
+from taxi.models import Driver, Manufacturer
+from taxi.templatetags.query_transform import query_transform
+
+
+class DriverCreatingFormTest(TestCase):
+
+    def test_driver_creation_form_is_valid(self):
+        form_data = {
+            "username": "new_user",
+            "password1": "poiuytre123456",
+            "password2": "poiuytre123456",
+            "license_number": "TST23456",
+            "first_name": "John",
+            "last_name": "Doe",
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+
+    def test_form_saves_new_driver_instance_successfully(self):
+        form_data = {
+            "username": "new_user",
+            "password1": "poiuytre123456",
+            "password2": "poiuytre123456",
+            "first_name": "John",
+            "last_name": "Doe",
+            "license_number": "TST23456",
+        }
+        form = DriverCreationForm(data=form_data)
+        if form.is_valid():
+            driver = form.save()
+            self.assertIsInstance(driver, Driver)
+            self.assertEqual(driver.username, form_data["username"])
+
+    def test_form_is_invalid_with_correct_license_number(self):
+        form_data = {
+            "username": "new_user",
+            "password1": "<PASSWORD>",
+            "password2": "<PASSWORD>",
+            "first_name": "John",
+            "last_name": "Doe",
+            "license_number": "tys12",

```

Comment:
Avoid duplicating code whenever possible.

For more information on this principle, read [here](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).